src/components/PopupWithForm.js

```diff
@@ -15,6 +16,14 @@ export default class PopupWithForm extends Popup {
     return this._modalValues;
   }
 
+  renderLoading(tyringToSave) {
+    if (tyringToSave) {
+      this._modalButton.textContent = "Saving...";
+    } else {
+      this._modalButton.textContent = "Save";
+    }
+  }
+
   close() {
     super.close();
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.